doc/releases/release-notes-2.5.rst

```diff
@@ -258,6 +258,8 @@ MCUBoot
   * Added reset of Cortex SPLIM registers before boot.
   * Fixesd build issue that occurs if CONF_FILE contains multiple file paths
     instead of single file path.
+  * Added watchdog feed on nRF dvices. See ``CONFIG BOOT_WATCHDOG_FEED`` option.
```

Comment:
```suggestion
  * Added watchdog feed on nRF devices. See ``CONFIG_BOOT_WATCHDOG_FEED`` option.
```

Reply:
corrected

Reply:
`CONFIG BOOT_WATCHDOG_FEED` still needs to be corrected. `_` after `CONFIG` is missing.